saleor/graphql/product/tests/mutations/test_product_bulk_create.py

```diff
@@ -1275,7 +1275,7 @@ def test_product_bulk_create_with_variants_and_invalid_stock(
     assert not data["results"][0]["product"]
     errors = data["results"][0]["errors"]
     assert errors
-    assert errors[0]["path"] == "variants.1.warehouses"
+    assert errors[0]["path"] == "variants.1.stocks.0.warehouse"
```

Comment:
I think we should mark `path` as a preview feature wherever it is used in 3.13, also in already released 3.13. These paths are changed now so if anyone used that already, it is breaking.

Reply:
The mutation is marked as "preview"; we just need to mention that path is changed in the breaking changes section in 3.14 changelog.

Reply:
A mutation that uses this field is marked as `Preview` so I added a note to 3.14 `Breaking changes` about more detailed paths for that mutation.